test/browser.mjs

```diff
@@ -54,6 +54,14 @@ test('concat', (t) => {
   t.alike(b.concat([b.from([1, 2, 3]), b.from([4, 5, 6])]), b.from([1, 2, 3, 4, 5, 6]))
 })
 
+test('concat with length', (t) => {
+  t.alike(b.concat([b.from([1, 2, 3]), b.from([4, 5, 6])], 5), b.from([1, 2, 3, 4, 5]))
+  t.alike(b.concat([b.from([1, 2, 3]), b.from([4, 5, 6], [7, 8, 9])], 5), b.from([1, 2, 3, 4, 5]))
+  t.alike(b.concat([b.from([1, 2, 3]), b.from([4, 5, 6])], 6), b.from([1, 2, 3, 4, 5, 6]))
+  const initValue = 0
```

Comment:
Any reason this binding?

Reply:
Not really, it was just my way of communicating that the last element of the 7-entry array is still the initial value. I'll remove it